themes/src/main/resources/theme/base/admin/resources/js/controllers/realm.js

```diff
@@ -965,6 +965,9 @@ module.controller('RealmIdentityProviderCtrl', function($scope, $filter, $upload
         for (var key in data) {
             $scope.identityProvider.config[key] = data[key];
         }
+        if (data["enabledFromMetadata"] !== undefined ) {
+            $scope.identityProvider.enabled = data["enabledFromMetadata"] == "true";
```

Comment:
`enabledFromMetadata` should be also removed from `$scope.identityProvider.config`.